src/main/java/org/mifos/connector/mockpaymentschema/api/BatchApi.java

```diff
@@ -41,4 +45,15 @@ public ResponseEntity<Object> getAuthorization(@PathVariable String batchId,
         }
         return new ResponseEntity<>(HttpStatus.ACCEPTED);
     }
+
+    @GetMapping(value = "/batches/{batchId}/summary", produces = MediaType.APPLICATION_JSON_VALUE)
+    public BatchDTO batchSummary(@PathVariable String batchId) {
+        return batchService.getBatchSummary(batchId);
+    }
+
+    @GetMapping(value = "/batches/{batchId}/detail", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
This API documentation should be upated. Please attach link to postman PR.
Also make sure that the validations and error handling is done.

ref: https://docs.google.com/spreadsheets/d/13TdoMUXtmV4MqwafbZu4_zvwOUhbO2WTs1UxZOLXHXI/edit#gid=0

Reply:
https://github.com/openMF/ph-ee-env-template/pull/411
I have added postman collection for this.